services/madoc-ts/src/extensions/enrichment/models.ts

```diff
@@ -0,0 +1,26 @@
+import { EnrichmentEntity, EnrichmentEntityType } from './authority/types';
+import { CaptureModelShorthand } from '../../frontend/shared/capture-models/types/capture-model-shorthand';
+
+export const entityTypeModel: CaptureModelShorthand<EnrichmentEntityType> = {
+  other_labels: { type: 'international-field', label: 'Display label' },
+  label: { type: 'text-field', label: 'Canonical label' },
+  description: { type: 'international-field', label: 'Description' },
+  image_url: { type: 'madoc-media-explorer', label: 'Image', valueAsString: true },
+  // other_data.
+  'other_data.example_data': { type: 'text-field', label: 'Example data' },
```

Comment:
IDE should give autocomplete for shorthand properties. For multiple items (lists) there are some tricks (see `__nested__` usage below). This should allow us to quickly expand the form for changing editorial.